spec/services/reporting/total_user_count_report_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+require 'csv'
+
+RSpec.describe Reporting::TotalUserCountReport do
+  let(:report_date) { Date.new(2021, 3, 1) }
+  let(:expected_report) do
+    [
+      ['All-time user count'],
+      [expected_count],
+    ]
+  end
+
+  subject(:report) { described_class.new(report_date) }
+
+  before { travel_to report_date }
```

Comment:
This is slightly copypasta; we don't assert the actual date here. But it nicely avoids the hassle I ran into previously where I had to set `created_at` to `Time.now - 1.minute` to work around some weirdness, so I left it with just following the existing pattern.